app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Sw</title>
+    <title>Gighub</title>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
 
   <body>
+    <header>
+      <h1></h1>
```

Comment:
@JosiMcClellan, Just out of curiosity, did you need to add the header tags to change the db name? Or was that just for funsies?